packages/sdk/src/conversation/conversation.ts

```diff
@@ -94,18 +95,78 @@ export class ConversationBot {
     if (options.adapter) {
       this.adapter = options.adapter;
     } else {
-      this.adapter = new BotFrameworkAdapter({
-        appId: process.env.BOT_ID,
-        appPassword: process.env.BOT_PASSWORD,
-      });
+      this.adapter = this.createDefaultAdapter(process.env.BOT_ID, process.env.BOT_PASSWORD);
+    }
```

Comment:
Can we also allow user to pass in the `appId` and `appPassword` to create the default adapter?

Reply:
Good suggestion. Added `adapterConfig` as well.